packages/superset-ui-demo/storybook/shared/components/ResizablePanel.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren, ReactNode } from 'react';
+import { ResizableBox, ResizableBoxProps } from 'react-resizable';
+
+import 'react-resizable/css/styles.css';
+
+export default function ResizablePanel({
+  children,
+  heading = undefined,
+  initialSize = { width: 500, height: 300 },
+  minConstraints = [100, 100] as [number, number],
+  panelPadding = 30,
```

Comment:
not sure about adding padding here though. I would prefer not to add because the components when used in Superset and other places do not have this padding. It may be misleading during the component development that there are paddings.

Reply:
The chart containers in Superset do have some padding. I was simulating the environment when charts are rendered in dashboard panels or on the explore page.

Just moved the paddings to `ResizableChartDemo`, so users specify `initialSize` for the panel, instead of the content in the panel (i.e. `SupersetChart`). Not sure which ways is less misleading though...